arch/arm/Kconfig

```diff
@@ -13,7 +13,7 @@ choice
 config ARCH_CHIP_A1X
 	bool "Allwinner A1X"
 	select ARCH_CORTEXA8
-	select ARCH_HAVE_FPU
+	select ARM_HAVE_NEON
```

Comment:
why change

Reply:
A1X support NEON, and ARM_HAVE_NEON will select ALL FPU relative confg.